definitions/npm/@entur/sdk_v1.x.x/flow_v0.25.x-/sdk_v1.x.x.js

```diff
@@ -0,0 +1,1032 @@
+type $entur$sdk$TransportMode =
```

Comment:
The inexact object syntax `...` was added in Flow 0.83.x, so it should be scoped to at least that version. 
We usually use 0.104.x, for when we run the codemod on the whole repo.

Reply:
I bumped the range to 0.83-